sunpy/map/mapbase.py

```diff
@@ -2049,7 +2049,11 @@ def plot(self, annotate=True, axes=None, title=True,
                               SunpyUserWarning)
 
         # Normal plot
-        plot_settings = copy.deepcopy(self.plot_settings)
+        try:
```

Comment:
I've been unable to get a dev MPL to work in my WSL2 setup so time to use the CI to test this.

Reply:
Turns out I can't read. I installed MPL master and it doesn't seem to error.